web/src/api/fbCalcAPI.ts

```diff
@@ -52,14 +51,15 @@ export interface FetchableFBCStation {
 }
 
 export async function postFBCStations(
+  date: string,
   fireBehaviorStations: FetchableFBCStation[]
-): Promise<FBCStation[]> {
+): Promise<FBCWeatherStationsResponse> {
```

Comment:
I missed this in the front end refactor; might want to rename to `FBC` qualifier to `FBA` for consistency:
`FBCWeatherStationsResponse` -> `FBAWeatherStationsResponse`
`postFBCStations` -> `postFBAStations`
`FetchableFBCStation` -> `FetchableFBAStation`
`FBCStation` -> `FBAStation`
`fbCalcAPI.ts` -> `fbaCalcAPI.ts`